### PR TITLE
{compiler}[system/system] ispc v1.12.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@furstj Any chance of a test report?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b40c1e3dd156cf967b7cdcf0130e26df for a full test report.

**Comment by furstj** [comment]
Test report by @furstj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
thinkpad - Linux UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz, Python 3.6.10
See https://gist.github.com/376291afcde78846eb15acc630dcdbfb for a full test report.

File: easybuild/easyconfigs/i/ispc/ispc-1.12.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+source_urls = [('http://sourceforge.net/projects/ispcmirror/files/v%(version)s', 'download')]
+sources = ['ispc-v%(version)s-linux.tar.gz']
+checksums = ['79decaef42bc5433ab30f5879eb91af14473a42e8b51ec727a97f61ee918497e']
+
+sanity_check_paths = {
+    'files': ["bin/ispc"],
+    'dirs': []
+}
+
+modextrapaths = {'PATH': ''}
```
Comment (boegel) [inline easybuild/easyconfigs/i/ispc/ispc-1.12.0.eb]: @furstj This is no longer needed now `ispc` is properly in a `bin/` subdirectory?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2410.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/22f2fec846abc37787ef2997a60346fb for a full test report.

File: easybuild/easyconfigs/i/ispc/ispc-1.12.0.eb (hunk 1 above)
Comment (furstj) [reply easybuild/easyconfigs/i/ispc/ispc-1.12.0.eb]: The modextrapaths is removed in latest commit

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2548.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/feb566a208d37eff15919d65c1d0fd6a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @furstj!

**Pull request merged**